sidebars.js

```diff
@@ -58,6 +58,28 @@ module.exports = {
     {
       "Svix Play": ["play"]
     },
+    {
+      "Svix Stream": [
```

Comment:
I think it should live in its own section of the docs. Maybe @svix-lucho can help here. Though essentially similar to have this section is essentially a completely doc site: https://docs.svix.com/receiving/introduction

We probably want to have a completely separate doc site for Stream, Ingest, and Dispatch (again, similar to the above), with an app selector.

Lucho, can you help with the app selector in the navbar or something? (Posthog did it well if memory serves)